server.js

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable no-console */
 import express from 'express';
+import './env.config';
+
+const PORT = process.env.MY_PORT || 3000;
 
 const app = express();
 
 app.get('/', (req, res) => {
   res.send('Hello world');
 });
 
-app.listen(3000, () => {
-  console.log('Listening on port 3000');
+app.listen(PORT, () => {
+  console.log(`server listening on: http://localhost:${PORT}`);
```

Comment:
replace that console .log to : process.stdout.write(`server listening on: http://localhost:${PORT});